host/3.0/buster/amd64/python/python38/python38.Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM python:3.8-slim-buster
+FROM mcr.microsoft.com/mirror/docker/library/python:3.8-buster
```

Comment:
Callout. Changed this image from slim to non-slim which will include more dependencies in the base image. Fine to keep as long as you are aware. 

Dependency set will only increase with this change so no risk of breaking apps.